app/inject.js

```diff
@@ -181,6 +181,23 @@ function init() {
             flash.setAttribute("wmode", "transparent");
         })();
 
+        // reload page if draft page
+        if (LoungeUser.userSettings.redirect === "1") {
+            if (document.body.textContent.indexOf("Item draft is under progress") !== -1) {
```

Comment:
Question: will this refresh a page if a user in a trade description has written 'Item draft is under progress'? :>